### DISCUSSION TITLE
The anatomy of KS6

### DISCUSSION THREAD

**Discussion by borg** [keystonejs/keystone#6580] [Event Q&A Q3 2021]
It seems the wires are more hidden on KS6, as in what is the relationship between KS6 and plain Node/Express server/Apollo? How come it's possible to add KS support to Next app with pretty much one dependency - seems like magic? There used to be a mono repo approach, which is now seemingly decoupled by default, opening up for the front end to be on a totally different domain. How does that work with shared sessions and cors? 

A more abstract schematic/philosophical view of what is core to KS and how it relates to those other core components (node/apollo/next), and why some of those choices were made, would be very welcomed. In particular for people who are new to the GraphQL eco system such an intro would be very helpful.

**Comment by raveling** [comment]
Good question @borg 
Tim gave a considered response in the recent community Q&A event:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=2079s
